File: solutions/java/src/fooddeliveryservice/Customer.java
package fooddeliveryservice;

import java.util.UUID;

public class Customer {
    private final String id;
    private final String name;
    private final String email;
    private final String phone;

    public Customer(String name, String email, String phone) {
        this.id = UUID.randomUUID().toString();
        this.name = name;
        this.email = email;
        this.phone = phone;
    }

    public String getId() {
        return id;
    }
}

File: solutions/csharp/fooddeliveryservice/Customer.cs
namespace FoodDeliveryService
{
    public class Customer
    {
        public string Id { get; }
        public string Name { get; }
        public string Email { get; }
        public string Phone { get; }

        public Customer(string id, string name, string email, string phone)
        {
            Id = id;
            Name = name;
            Email = email;
            Phone = phone;
        }
    }
}